tfhe/src/boolean/engine/bootstrapping.rs

```diff
@@ -80,8 +80,8 @@ impl Memory {
 /// * `key_switching_key` - a public key, used to perform the key-switching operation.
 #[derive(Clone)]
 pub struct ServerKey {
-    pub(crate) bootstrapping_key: FourierLweBootstrapKeyOwned,
-    pub(crate) key_switching_key: LweKeyswitchKeyOwned<u32>,
+    pub bootstrapping_key: FourierLweBootstrapKeyOwned,
+    pub key_switching_key: LweKeyswitchKeyOwned<u32>,
 }
```

Comment:
we can add a function in impl ServerKey that would be `pub fn bootstrapping_key_size_elements(&self) -> usize` or even `pub fn bootstrapping_key_size_bytes(&self) -> usize` that basically does

```rust
        self.bootstrapping_key.as_ref().len() * std::mem::size_of::<concrete_fft::c64>()
```

what do you think ?

Reply:
and then an equivalent function for the ksk

Reply:
Yes, I like the idea. But in this case, we should do it for Shortint keys too right ?

Reply:
yes that's fine by me !